src/ProjectManager.ts

```diff
@@ -34,7 +33,7 @@ class ProjectManager {
     if (!this.repoParser) {
       this.repoParser = await getRepoParser(this.workspace.uri.fsPath, this.log, jestPath);
       if (this.repoParser) {
-        this.disposables.push(this.repoParser.projectChange(this.handleProjectChange));
+        this.disposables.push(this.repoParser.projectChange(event => this.handleProjectChange(event)));
```

Comment:
I think we should allow the original form in cases like this.  It is more verbose and IMHO doesn't add anything of value.  Can we alter the linting rules appropriately?

Reply:
Rule triggered: https://github.com/typescript-eslint/typescript-eslint/blob/v3.7.1/packages/eslint-plugin/docs/rules/unbound-method.md

While in the case above you are right, it will not change the outcome, it's not the case if method is passed as an argument to different context. This kind of a bug can be hard to find for beginner programmers.

Reply:
Ok cool, it's `this` again!  Happy to have this rule then.  I'll have to adjust my style.